classes/journal/JournalDAO.inc.php

```diff
@@ -418,6 +422,67 @@ function resequenceJournals() {
 	function getInsertJournalId() {
 		return $this->getInsertId('journals', 'journal_id');
 	}
+	
+	/**
+	 * Get journals by setting.  Backported from master pkp-lib's ContextDAO.
+	 * @param $settingName string
+	 * @param $settingValue mixed
+	 * @param $contextId int
+	 * @return DAOResultFactory
+	 */
+	function getBySetting($settingName, $settingValue, $contextId = null) {
+		$params = array($settingName, $settingValue);
+		if ($contextId) $params[] = $contextId;
+
+		$result = $this->retrieve(
+			'SELECT * FROM ' . $this->_getTableName() . ' AS c
+			LEFT JOIN ' . $this->_getSettingsTableName() . ' AS cs
+			ON c.' . $this->_getPrimaryKeyColumn() . ' = cs.' . $this->_getPrimaryKeyColumn() .
+			' WHERE cs.setting_name = ? AND cs.setting_value = ?' .
```

Comment:
Frankly, that's the kind of tweak I'd commit directly to master and skip the PR hassle entirely. Since you're just back-porting, and it's entirely a stylistic thing (I think any DB optimizer will catch this), best just leave it as is for consistency, I think.